README.md

```diff
@@ -54,20 +110,25 @@ pgdbname=dcrdex_testnet
 pgpass=dexpass
 ```
 
-The **app data directory** (*~/.dcrdex/*) is the default location used by the
-DEX server, but can be customized with a command-line argument.
+The *~/.dcrdex/* is the default **app data directory** location used by the
+DEX server, but can be customized with the `--appdata` command-line argument.
 
 Master public keys are network-specific, so **dcrd** and **dcrwallet** will need
 to be run on the same network on which the server will be run.
```

Comment:
For dcrdex server setup, technically don't need **dcrwallet** to be running, at least not on the same machine as the server. I'm thinking a cold wallet is a better idea for collecting fees.